Update free-courses-hi.md (#9867)
EbookFoundation/free-programming-books: courses/free-courses-hi.md

* [Github \| All about Git and GitHub](https://www.youtube.com/watch?v=77b2lVHHZqI) - Anuj Bhaiya
* [GitHub Tutorial in Hindi](https://youtube.com/playlist?list=PLVdoaEL574VBxxcGQmTjxS-JoP5rKV8Wi) - Be A Programmar
* [GitHub with Visual Studio (Hindi)](https://www.youtube.com/playlist?list=PLbGui_ZYuhigWA1mNWzwErSBIZvgOJbNc) - Rajesh Kumar, Geeky Shows

> * [Complete Git and Github tutorial](https://youtu.be/Ez8F0nW6S-w?si=y4V7MYwZ7bnjduCz) - Apna College

Reviewer: ```suggestion
* [Complete Git and Github Tutorial for Beginners](https://www.youtube.com/watch?v=Ez8F0nW6S-w) - Apna College
```
title from resourse

* [MongoDB Tutorial for Beginners (Hindi)](https://www.youtube.com/playlist?list=PLgPJX9sVy92xUxpTFgAOSBHdBwIdxav39) - CS Geeks
* [MongoDB tutorial in hindi](https://www.youtube.com/playlist?list=PLolI8AY2AS9aaE4Vx0adwfwUh3XiuVewX) - Code Improve
* [MongoDB Tutorial in Hindi 2022](https://www.youtube.com/playlist?list=PLQDioScEMUhkcqbgJ4ap2k4zg3sT_-Bbc) - Programming Experience

> * [MongoDB full course](https://youtube.com/playlist?list=PLA3GkZPtsafZydhN4nP0h7hw7PQuLsBv1&si=fj9OK4DshAlDpJd_) - Engineering Digest

Reviewer: ```suggestion
* [MongoDB full course](https://www.youtube.com/playlist?list=PLA3GkZPtsafZydhN4nP0h7hw7PQuLsBv1&si=fj9OK4DshAlDpJd_) - Engineering Digest
```

> * [MongoDB full course](https://www.youtube.com/playlist?list=PLA3GkZPtsafZydhN4nP0h7hw7PQuLsBv1&si=fj9OK4DshAlDpJd_) - Engineering Digest

Reviewer: ```suggestion
* [MongoDB Playlist in Hindi](https://www.youtube.com/playlist?list=PLA3GkZPtsafZydhN4nP0h7hw7PQuLsBv1&si=fj9OK4DshAlDpJd_) - Engineering Digest
```
